jkube-kit/build/service/docker/src/test/java/org/eclipse/jkube/kit/build/service/docker/access/BuildConfigTest.java

```diff
@@ -18,80 +18,79 @@
 import java.util.Map;
 
 import org.eclipse.jkube.kit.common.JsonFactory;
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNull;
+import static org.assertj.core.api.Assertions.assertThat;
 
 /**
  * @author roland
  * @since 03/01/17
  */
-public class BuildConfigTest {
+class BuildConfigTest {
 
     @Test
-    public void empty() {
+    void empty() {
         BuildOptions opts = new BuildOptions();
-        assertEquals(0, opts.getOptions().size());
+        assertThat(opts.getOptions()).isEmpty();
     }
 
     @Test
-    public void forcerm() {
-        BuildOptions opts = new BuildOptions().forceRemove(false);
-        assertEquals(0, opts.getOptions().size());
-        opts = new BuildOptions().forceRemove(true);
-        assertEquals("1", opts.getOptions().get("forcerm"));
+    void forcerm() {
+        BuildOptions optsWithoutForcerm = new BuildOptions().forceRemove(false);
+        assertThat(optsWithoutForcerm.getOptions()).isEmpty();
+        BuildOptions optsWithForcerm = new BuildOptions().forceRemove(true);
+        assertThat(optsWithForcerm.getOptions()).containsEntry("forcerm", "1");
```

Comment:
**No need to change**, but just as a note.

This test is actually testing 2 different things. The best approach would be to separate these tests into 2 separate tests (or a parameterized test). This way, in case of failure, you'd actually know what is failing. In addition, if the first part of the test fails, you won't know if the second part fails too because it will never run.

Reply:
Thanks for the note, I'll remember it.